Support object rest operator

### What is this bug about?

It's currently not possible to use the object rest operator in a wolkenkit application.

### What steps are needed to reproduce the bug?

-  init a sample application
- Add the following code to the `sampleCommand`

```javascript
async sampleCommand (sampleAggregate, command) {
    const sampleObject = { ...command.data };
    
    sampleAggregate.events.publish('sampleEvent', {
      // ...
    });
  }
```
- run `wolkenkit start`

### What is the expected result?

The application should start.

### What is the actual result?

The CLI fails to start with the following output:

```shell
  Starting the application...
  
/Users/matthiaswagler/Dev/tnw/temp/destructuring-test/server/writeModel/sampleContext/sampleAggregate.js
  17:28  error  Parsing error: Unexpected token ..

  1 problem (1 error, 0 warnings)
✗ Failed to start the application.

```

### What else should we know?

Seems like the linter which checks the application before starting it, does not support the spread operator for objects. It's available [since Node 8.6](https://node.green/#ES2018-features-object-rest-spread-properties). So it should be save to enable.

--- a/package.json
+++ b/package.json
@@ -75,7 +75,7 @@
   },
   "devDependencies": {
     "assertthat": "2.0.1",
-    "dir-compare": "1.6.0",
+    "dir-compare": "1.7.1",
     "measure-time": "3.1.1",
     "p-queue": "3.0.0",
     "roboter": "3.0.4",

--- a/src/wolkenkit/commands/shared/validateCode.js
+++ b/src/wolkenkit/commands/shared/validateCode.js
@@ -17,7 +17,7 @@ const validateCode = function ({ directory }, progress) {
   const cliEngine = new eslint.CLIEngine({
     envs: [ 'node', 'es6' ],
     parserOptions: {
-      ecmaVersion: 2017,
+      ecmaVersion: 2019,
       ecmaFeatures: {}
     },
     rules: {

